vignettes/epipredict.Rmd

```diff
@@ -0,0 +1,360 @@
+---
+title: "Get started with epipredict"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{epipredict}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  echo = TRUE,
+  collapse = TRUE,
+  comment = "#>",
+  out.width = "100%")
+```
+
+```{r setup, message=FALSE}
+library(dplyr)
+library(parsnip)
+library(workflows)
+library(recipes)
+library(epiprocess)
+# remotes::install_github("cmu-delphi/epipredict")
+library(epipredict)
+```
+
+
+# Goals for the package
+
+At a high level, our goal with `epipredict` is to make running simple Machine Learning / Statistical forecasters for epidemiology easy. However, this package is extremely extensible, and that is part of the utility. Our hope is that it is easy for users with epi training and some statistics to fit baseline models while still allowing those with more nuanced statistical understanding to create complicated specializations within the same framework.
+
+Serving both populations is the main motivation for our efforts, but at the same time, we have tried hard to make it useful.
```

Comment:
What is the "both populations" you are speaking of?

Reply:
Those mentioned in the previous paragraph (epi training + some stats vs. more nuanced statistical understanding). Not clear enough?

Reply:
Alright, now I understand.